### PR TITLE
SEO-178962-JavaScript-Charts-Improvement

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @Gowri-V-L,
Please find changes made in the md file.
Regards,
Scholar.

### CODE REVIEW

File: ej2-javascript/diagram/ts/symbol-palette.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Symbol palette in ##Platform_Name## Diagram control | Syncfusion
-description: Learn here all about Symbol palette in Syncfusion ##Platform_Name## Diagram control of Syncfusion Essential JS 2 and more.
+title: Symbol palette in JavaScript Diagram Control | Syncfusion
+description: Learn here all about symbol palette in Syncfusion Essential JavaScript Diagram control, its elements and more.
```
Comment (Gowri-V-L) [inline ej2-javascript/diagram/ts/symbol-palette.md:4]: Symbol palette - Its a control name so it should be with caps

Comment (Scholar-jaruha) [reply ej2-javascript/diagram/ts/symbol-palette.md]: changed

Comment (Scholar-jaruha) [reply ej2-javascript/diagram/ts/symbol-palette.md]: changed

**Review by Gowri-V-L — APPROVED** [review]
changes are fine

**Pull request merged**